src/screens/EditOrderScreen.tsx

```
@@ -20,33 +23,89 @@ const EditOrderScreen: React.FC<EditOrderScreenProps> = ({
+    customerName: '',
+    title: '',
     description: '',
-    priority: t('orders.priorities.normal'),
+    priority: 'Normal' as 'Low' | 'Normal' | 'High' | 'Urgent',
     category: '',
-    status: t('orders.statuses.open'),
+    status: 'Open' as any,
   });
+  const [customers, setCustomers] = useState<User[]>([]);
+  const [showCustomerModal, setShowCustomerModal] = useState(false);
+  const [originalOrder, setOriginalOrder] = useState<Order | null>(null);
```

ESLint

[error] 'originalOrder' is assigned a value but never used. Allowed unused elements of array destructuring patterns must match /^_/u.

(@typescript-eslint/no-unused-vars)